modules/op-release-notes-1-0.adoc

```diff
@@ -72,6 +72,45 @@ For this release, use flags `disable-home-env-overwrite` and `disable-working-di
 == Known issues
 * If you are upgrading from an older version of {pipelines-title}, you must delete your existing deployments before upgrading to {pipelines-title} version 1.0. To delete an existing deployment, you must first delete Custom Resources and then uninstall the {pipelines-title} Operator. For more details, see the uninstalling {pipelines-title} section.
 * Submitting the same `v1alpha1` Tasks more than once results in an error. Use `oc replace` instead of `oc apply` when re-submitting a `v1alpha1` Task.
+* The `buildah` ClusterTask does not work when a new user is added to a container.
++
+When the Operator is installed, the `--storage-driver` flag for the `buildah` ClusterTask is not specified, therefore the flag is set to its default value. In some cases, this causes the storage-driver to be set incorrectly. When a new user is added, the incorrect storage-driver results in the failure of the `buildah` ClusterTask with the following error:
```

Comment:
Do you mean /`storage-driver/` to refer to a specific parameter value or a generic reference to a storage driver? This scans like you either need to enclose the term in backticks or remove the hyphen.